repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/ipuser/IpUserTest.groovy

```diff
@@ -89,4 +93,23 @@ class IpUserTest extends ReposeValveTest {
         !group.contains("local-lan-ip;q=0.6")
     }
 
+    def "should fail to start with bad config"() {
```

Comment:
I was thinking about it, but this test suite is written in a very odd manner and doesn't follow our normal patterns anyway. That said, I don't have any heartache splitting this out for consistency. I guess this is as good a place as any to start making this thing right.